jwst/regtest/test_nirspec_ifu.py

```diff
@@ -104,6 +104,7 @@ def run_spec3_multi(jail, rtdata_module):
     return rtdata
 
 
+@pytest.mark.skip(reason="single exposure processing is redundant with calspec2")
```

Comment:
Still on the wrong test.  Next test down.  `test_spec3`

Reply:
D'oh!! My brain is fried ...